beacon_node/http_api/src/publish_blocks.rs

```diff
@@ -86,6 +86,27 @@ pub async fn publish_block<T: BeaconChainTypes>(
 
             Ok(())
         }
+        Err(BlockError::BlockIsAlreadyKnown) => {
+            info!(
+                log,
+                "Block from HTTP API already known";
+                "block" => ?block.canonical_root(),
+                "slot" => block.slot(),
+            );
+            Ok(())
+        }
+        Err(BlockError::RepeatProposal { proposer, slot }) => {
+            warn!(
+                log,
+                "Block ignored due to repeat proposal";
+                "msg" => "this can happen a VC uses fallback BNs. \
```

Comment:
```suggestion
                "msg" => "this can happen when a VC uses fallback BNs. \
```

typo back from the dead :stuck_out_tongue_closed_eyes: